BUILDING.md

```diff
@@ -202,6 +202,9 @@ Prerequisites:
   [Git for Windows](http://git-scm.com/download/win) includes Git Bash
   and tools which can be included in the global `PATH`.
 
+*Note:* All prerequisites can be easily installed by following
+[this bootstrapping guide](https://github.com/nodejs/node/blob/master/tools/bootstrap/README.md#Windows).
```

Comment:
Maybe not move this to the top of the list and drop the `#Windows` off the end? That way it'll work when we add other platforms.

Reply:
Sorry, I don't understand what you mean

Reply:
Move this line to L83, and change it to:

```markdown
All prerequisites can be easily installed by following
[this bootstrapping guide](https://github.com/nodejs/node/blob/master/tools/bootstrap/README.md).
```